Tests/conf.json

```
@@ -5624,6 +5624,10 @@
         {
             "integrations": "Netcraft V2",
             "playbookID": "Test - Netcraft"
+        },
+                {
```

Please pay attention to the indents
```suggestion
        {
```